sdk/keyvault/Azure.Security.KeyVault.Certificates/src/Issuer.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+
+namespace Azure.Security.KeyVault.Certificates
+{
+    public class Issuer : IssuerBase
+    {
+        public readonly string Self = "self";
+        public readonly string Unknown = "unknown";
+
+        public string AccountId { get; set; }
+        public string Password { get; set; }
+        public string OrganizationId { get; set; }
+        public string Email { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Phone { get; set; }
```

Comment:
From the API docs mentioned [here](https://docs.microsoft.com/en-us/rest/api/keyvault/getcertificateissuer/getcertificateissuer#organizationdetails) looks like`admin_details` is a list. 
But if it is a list we probably cannot flatten it out this way, maybe?
Talked with @schaabs on this, he said he can confirm with the service team on the usage and then can decide further on this.

Reply:
Great. Thanks! I will wait on that then